node/src/chain_spec.rs

```diff
@@ -217,8 +219,11 @@ pub fn foucoco_config() -> FoucocoChainSpec {
 				// initial collators.
 				invulnerables.clone(),
 				signatories.clone(),
+				//todo: is it okay that only sudo is authorized?
+				vec![(sudo_account.clone(), "Sudo".as_bytes().to_vec())],
```

Comment:
In theory: no. But in practice, we will have to change this anyways once the DIA pallet is implemented. So let's think about it again once that happens.